test/fixtures/state-saved-v1-3.js

```diff
@@ -283,7 +285,10 @@ export const savedStateV1 = {
             },
             enabled: true
           },
-          brush: {size: 0.5, enabled: false}
+          brush: {size: 0.5, enabled: false},
+          geocoder: {
```

Comment:
the saved state should not be changed, the merged config should have geocoder set to `{enablced: false}`